### PR TITLE
fix: move box-sizing property to global styles

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-82](https://klau.atlassian.net/browse/FSU-82)

## Describe your changes

✨ ✨ This is a special PR for @paulgrym, just to prove that I'm not working on the project on Mondays only 😅

It removes local box-sizing properties added to the Card recipe and adds them to global styles.
They're just inline styles for now, but we can create a global styles file as soon as we need more of them.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by AmandaKr — APPROVED** [review]
🦸‍♀️

**Review by nata-kab — APPROVED** [review]
Looks super clean and works perfectly 🙌

**Review by paulgrym — COMMENTED** [review]
Nice job 👏

File: packages/frontend/app/pages/_app.jsx [hunk 1]
```diff
@@ -11,6 +10,13 @@ const MyApp = ({ Component, pageProps }) => (
     <Head />
     <ThemeProvider theme={theme}>
       <Normalize />
+      <Global
+        styles={css`
+          * {
```
[the remark this answers is not in the document]
Comment (paulgrym) [reply packages/frontend/app/pages/_app.jsx:15]: oh and thank you for special dedication 🤣 Now I can see that you are unpredictable 😁

File: packages/design-system/.storybook/preview.js [hunk 2]
```diff
@@ -20,6 +20,13 @@ export const decorators = [
   (Story) => (
     <ThemeProvider theme={theme}>
       <Normalize />
+      <Global
+        styles={css`
+          * {
```
Comment (paulgrym) [inline packages/design-system/.storybook/preview.js:25]:
```suggestion
          *, ::before, ::after{
```

Comment (paulgrym) [reply packages/design-system/.storybook/preview.js:25]: I've checked and if we want box-sizing to apply to before and after elements we need to add this to our code😄

**Review by paulgrym — APPROVED** [review]
⭐

**Pull request merged**